test/test_speed_v_torch.py

```diff
@@ -221,13 +221,12 @@ def test_conv2d(self):
           img_size = 34
           torch_dat = torch.rand(bs, in_chans, img_size, img_size).to(torch_device)
           torch_conv = torch.nn.Conv2d(in_chans, out_chans, 3, bias=None).to(torch_device)
-
           tiny_dat = Tensor(torch_dat.cpu().numpy())
           tiny_conv = Conv2d(in_chans, out_chans, 3, bias=None)
           tiny_conv.weight = Tensor(torch_conv.weight.detach().cpu().numpy())
 
-          def f1(torch_dat): return torch_conv(torch_dat)
-          def f2(tiny_dat): return tiny_conv(tiny_dat).realize()
+          def f1(td, torch_conv=torch_conv): return torch_conv.__call__(td)
```

Comment:
Why is this needed here and not on line 212?

Reply:
the linter is checking for when variables in closures are defined in loops